File: user-service/src/main/java/com/github/manosbatsis/services/user/rest/ResourceSizeAdvice.java
package com.github.manosbatsis.services.user.rest;

import org.springframework.web.bind.annotation.ControllerAdvice;
import org.springframework.web.bind.annotation.RestController;

@ControllerAdvice(annotations = RestController.class)
public class ResourceSizeAdvice extends com.github.manosbatsis.lib.core.rest.ResourceSizeAdvice {}

File: user-service/src/main/kotlin/com/github/manosbatsis/services/user/rest/ResourceSizeAdvice.kt
package com.github.manosbatsis.services.user.rest

import com.github.manosbatsis.lib.core.rest.ResourceSizeAdvice
import org.springframework.web.bind.annotation.ControllerAdvice
import org.springframework.web.bind.annotation.RestController

@ControllerAdvice(annotations = [RestController::class])
class ResourceSizeAdvice : ResourceSizeAdvice()
